tsdb/wal/wal.go

```diff
@@ -767,7 +767,7 @@ type segmentRef struct {
 func listSegments(dir string) (refs []segmentRef, err error) {
 	files, err := ioutil.ReadDir(dir)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("list segments in dir %s: %s", dir, err)
```

Comment:
Any reasons why we are not using `errors.Wrapf()`?

Reply:
thanks, pkg `errors` is better.